.github/actions/create-kubewarden-cluster/action.yaml

```diff
@@ -0,0 +1,64 @@
+name: "Setup Kubewarden cluster"
+description: "Create a K3d cluster with Kubewarden installed with Telemetry infrastructure installed"
+author: "Kubewarden"
+inputs:
+  controller-image-repository:
+    description: "Define the controller container image repository"
+    type: string
+    required: false
+    default: ""
+  controller-image-tag:
+    description: "Define the controller container image tag"
+    required: false
+    default: ""
+  load-local-image:
+    description: "Load the image used in the deployment from local artifact"
+    required: false
+    default: false
+runs:
+  using: "composite"
+  steps:
+      - uses: AbsaOSS/k3d-action@v1.5.0
+        name: "Create Kubernetes cluster"
+        with:
+          cluster-name: "e2e-test-cluster"
+      - name: "Download local image"
+        if: ${{ inputs.load-local-image }}
+        uses: actions/download-artifact@v2
+        with:
+          name: kubewarden-controller-image-${{inputs.controller-image-tag}}
+          path: /tmp
+      - name: "Load local image"
+        if: ${{ inputs.load-local-image }}
+        run: |
+          k3d image import /tmp/kubewarden-controller-image-${{inputs.controller-image-tag}}.tar -c e2e-test-cluster
+        shell: bash
+      - name: "Install cert-manager"
+        run: kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml
+        shell: bash
+      - name: "Wait cluster to be ready"
+        run: kubectl wait --for=condition=Ready nodes --all
+        shell: bash
+      - uses: azure/setup-helm@v1
+        name: "Install Helm"
+        with:
+          version: 'latest' # default is latest stable
+        id: install
+      - name: "Install cert-manager"
+        run: kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml
+        shell: bash
+      - name: "Wait for cert-manager be ready"
+        run: kubectl wait --for=condition=Available deployment --timeout=3m -n cert-manager --all
+        shell: bash
```

Comment:
This is a line copied from our docs just for the wait `cert-manager` to be ready before continuing the installation. Avoiding some possible weird problems due to missing components. To avoid one additional step, I'll move the command to the previous command. Sounds good?